src/main/java/com/keshavarzi/airplanestats/model/Trip.java

```diff
@@ -41,4 +56,22 @@ public class Trip {
 
     @Column(name = "trip_end", table = "trip")
     private ZonedDateTime tripEnd;
+
+//    @Nonnull
+//    public UserEntity getUserEntity() throws CloneNotSupportedException {
+//        return this.userEntity.clone();
+//    }
+//
+//    public void setUserEntity(@Nonnull UserEntity userEntity) throws CloneNotSupportedException {
+//        this.userEntity = userEntity.clone();
+//    }
+//
+//    public Collection<RouteEntity> getRoutes() {
+//        return List.copyOf(this.routeEntities);
+//    }
+//
+//    public void setRoutes(Collection<RouteEntity> routeEntities) {
+//        this.routeEntities = List.copyOf(routeEntities);
+//    }
```

Comment:
Dead code? Remove

Reply:
I think I will eventually use it.  Will add a `TODO` for myself